src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1217,6 +1217,10 @@ def set_up_existing_container_update(self):
             # Fails if all containers are removed
             while (len(self.new_containerapp["properties"]["template"]["containers"]) > 1):
                 self.new_containerapp["properties"]["template"]["containers"].pop()
+            if not self.get_argument_name():
```

Comment:
I'm just curious: is it even possible to have a container app without a name?

Reply:
Don't think it is possible to update a container app without the container app name. I'll remove this validation if it does not make sense for it since we are already checking for required arguments initially.